fdbserver/TLogServer.actor.cpp

```diff
@@ -1938,8 +1943,10 @@ Future<Void> tLogPeekMessages(PromiseType replyPromise,
 			rep.onlySpilled = false;
 
 			if (reqSequence.present()) {
-				auto& trackerData = logData->peekTracker[peekId];
-				auto& sequenceData = trackerData.sequence_version[sequence + 1];
+				auto [trackerIt, _] = logData->peekTracker.try_emplace(peekId);
```

Comment:
Is there any benefit to modifying the existing code to use "try_emplace() here? Is there any micro-optimization you are doing here that I am missing?

Reply:
see above. its to keep the access patterns consistent.